front/src/applications/operationalStudies/views/Scenario/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -89,7 +92,6 @@ const getTrainrunSectionsByTrainrunId = (
   const sections = netzgrafikDto.trainrunSections.filter(
     (section) => section.trainrunId === trainrunId
   );
-
```

Comment:
Nit: can we keep this empty newline? It's unrelated to this PR, and I like to keep space between logically separate blocks of code.